fetch.bs

```diff
@@ -7305,11 +7315,16 @@ constructor steps are:
  <li><p>Let <var>inputOrInitBody</var> be <var>initBody</var> if it is non-null; otherwise
  <var>inputBody</var>.
 
+ <li><p>If <var>init</var>["{{RequestInit/duplex}}"] is "full", then throw a {{TypeError}}.
```

Comment:
This will happen automatically as any unknown value ends up throwing so we can remove this step.

Reply:
Done.